subsys/bluetooth/services/ias/ias.c

```diff
@@ -31,12 +31,6 @@ LOG_MODULE_REGISTER(ias);
 #define IAS_ALERT_LEVEL_PERM BT_GATT_PERM_WRITE
```

Comment:
I see that this files was moved to an `ias` directory. 

Are we sure we want that?
@jhedberg @alwa-nordic Should we keep the `services` directory flat, or have these sub-directories?

Reply:
I've taken the pattern set up by OTS here in that matter.